### PR TITLE
Task about me section

### PR DESCRIPTION
**Pull request by milaferron**
About Me section ready for review. I will need to send my background video to you on slack.

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -136,12 +165,57 @@ footer {
...
+
+.section-header {
+    padding-top: 10px;
+    height: 250px;
+}
+
+.front-text {
+    color: #ffffff;
+    font-family: 'Muli', sans-serif;
+    font-weight: 700;
```
Comment (fredplatt) [inline css/style.css]: Might not be necessary - Turning off in inspector didn't seem to make any difference

File: css/style.css [hunk 2]
```diff
@@ -136,12 +165,57 @@ footer {
...
+    font-size: 4rem;
+    z-index: 2;
+    position: relative;
+    top: 55px;
+    text-transform: uppercase;
+}
+
+.shadow-text {
+    font-family: 'Covered By Your Grace', cursive;
+    color: #AA6373;
```
Comment (fredplatt) [inline css/style.css]: Might not be necessary - Turning off in inspector didn't seem to make any difference

File: css/style.css [hunk 3]
```diff
@@ -136,12 +165,57 @@ footer {
...
+    opacity: 0.3;
+    font-size: 7rem;
+    position: relative;
+    top: -173px;
+    z-index: 1;
```
Comment (fredplatt) [inline css/style.css:221]: Might not be necessary - Turning off in inspector didn't seem to make any difference - I've not used z index yet, but maybe if you've specified z2 for the front text, it assumes this one to be z1?

File: css/style.css [hunk 4]
```diff
@@ -105,6 +102,36 @@ header {
...
+
+.about-wrapper .scroll-icon {
+    color: #AA6373;
+    position: absolute;
+    bottom: -590px;
+    left: 50%;
+}
+
+.section-main {
+    width: 100%;
```
Comment (fredplatt) [inline css/style.css]: Might not be necessary - Turning off in inspector didn't seem to make any difference. It looks as though it's set to display: block so maybe that's doing 100% width for you already.

**Review by fredplatt — COMMENTED** [review]
Mostly based on flipping switches on inspector! Otherwise just a couple of places with extra lines in css - 152 + 153, and 194 + 195.

File: css/style.css (hunk 3 above)
Comment (milaferron) [reply css/style.css:221]: I was told by Mike that it is good practice to be explicit when on all elements when using z-index

File: css/style.css [hunk 5]
```diff
@@ -170,6 +166,15 @@ footer {
         /*width: 100%;*/
         float: none;
     }
+
+    .about-wrapper {
+        height: 111vh;
+    }
+
```
Comment (fredplatt) [inline css/style.css:173]: Another line to delete! Didn't spot last time, apologies :)

File: css/style.css [hunk 6]
```diff
@@ -211,11 +214,11 @@ footer {
...
     font-family: 'Covered By Your Grace', cursive;
-    color: #AA6373;
     opacity: 0.3;
     font-size: 7rem;
     position: relative;
     top: -173px;
     z-index: 1;
 }
 
+
```
Comment (fredplatt) [inline css/style.css:224]: Another line to delete! Didn't spot last time, apologies :)

**Review by fredplatt — APPROVED** [review]

**Pull request merged**